app/components/SetClaimConditions/SetClaimConditions.tsx

```
@@ -0,0 +1,315 @@
+            Select Payment Currency
+          </label>
+          <select
+            className="w-full rounded-md border px-3 py-2 text-slate-800 focus:outline-none focus:ring-2 focus:ring-blue-500 dark:bg-slate-200"
+            id="currency"
+            {...register('currency', { required: true })}
+            aria-invalid={formState.errors.currency ? 'true' : 'false'}
+          >
+            <option value="">-- Select Currency --</option>
+            {Object.keys(claimConditionsOptions.currency).map((k, i) => (
+              <option
+                key={i}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)